dvc/updater.py

```diff
@@ -123,14 +123,13 @@ def _notify(self):
     def _get_update_instructions(self):
         instructions = {
             "pip": "Run {yellow}pip{reset} install dvc {blue}--upgrade{reset}",
-            "yum": "Run {yellow}yum{reset} update dvc",
-            "yay": "Run {yellow}yay{reset} {blue}-S{reset} dvc",
-            "formula": "Run {yellow}brew{reset} upgrade dvc",
-            "apt": (
+            "rpm": "Run {yellow}yum{reset} update dvc",
+            "osxpkg": "Run {yellow}brew{reset} upgrade dvc",
```

Comment:
`osxpkg` is not the same as brew formula. We need to add build.py with `PKG = 'formula'` or `PKG = "brew"`(this one is even better :slightly_smiling_face: )  to our homebrew formula https://github.com/iterative/homebrew-dvc/blob/master/Formula/dvc.rb (after that we'll submit the same changes to homebrew-core).

Reply:
I think we could also `echo` somewhere there, need to take a closer look.

Reply:
I'll include that into our and homebrew-core formulas after this PR is merged.

Reply:
For the record: Added https://github.com/iterative/homebrew-dvc/commit/68d0f027b5b16f2d383991c51727d436674590f5 . Will play around with it and then release a new version and submit it to homebrew-core.